### ISSUE TITLE
[BUG] - Issue with Rancher Monitoring Prometheus Node Exporter Pod

### ISSUE THREAD

**Issue by skanakal** [rancher/rancher#46181]
**Rancher Server Setup**
- Rancher version: rancher version v2.9.0-alpha7 (c5733e1ae)
- Installation option (Docker install/Helm Chart): Helm
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): RKE2
- Proxy/Cert Details: Airgap Installation

**Information about the Cluster**
- Kubernetes version: v1.29.6+rke2r1
- Cluster Type (Local/Downstream): Local
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider):

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
  - If custom, define the set of permissions: Admin
 

**Describe the bug**

Issue with Rancher Monitoring Prometheus Node Exporter Pod on v2.9.0-alpha7 (airgap)
In the airgap installation, I noticed an issue with the Rancher Monitoring Prometheus Node Exporter pod in the cattle-monitoring-system namespace, it's trying to pull the wrong image:

**To Reproduce**

Install `rancher-monitoring-104.0.0-rc1+up57.0.3` chart on an airgap rancher server

**Result**
```
root@ip-10-0-13-92:~/rancher# kubectl get pods -n cattle-monitoring-system rancher-monitoring-prometheus-node-exporter-tzstb -o jsonpath='{.status.containerStatuses}' | jq
[
  {
    "image": "xxxxxxxx.us-east-2.compute.amazonaws.com/xxxxxxxx.us-east-2.compute.amazonaws.com/rancher/mirrored-prometheus-node-exporter:v1.3.1",
    "imageID": "",
    "lastState": {},
    "name": "node-exporter",
    "ready": false,
    "restartCount": 0,
    "started": false,
    "state": {
      "waiting": {
        "message": "Back-off pulling image \"xxxxxxxx.us-east-2.compute.amazonaws.com/xxxxxxxx.us-east-2.compute.amazonaws.com/rancher/mirrored-prometheus-node-exporter:v1.3.1\"",
        "reason": "ImagePullBackOff"
      }
    }
  }
]
root@ip-10-0-13-92:~/rancher# 
root@ip-10-0-13-92:~/rancher# kubectl -n cattle-system exec -it rancher-7d8bfb5976-btqq5 -- rancher -v
rancher version v2.9.0-alpha7 (c5733e1ae)
root@ip-10-0-13-92:~/rancher# 

root@ip-10-0-13-92:~/rancher# helm ls -n cattle-monitoring-system
NAME                  	NAMESPACE               	REVISION	UPDATED                                	STATUS  	CHART                                      	APP VERSION
rancher-monitoring    	cattle-monitoring-system	1       	2024-07-16 12:25:11.614883033 +0000 UTC	deployed	rancher-monitoring-104.0.0-rc1+up57.0.3    	v0.72.0    
rancher-monitoring-crd	cattle-monitoring-system	1       	2024-07-16 12:25:05.402031705 +0000 UTC	deployed	rancher-monitoring-crd-104.0.0-rc1+up57.0.3	           
root@ip-10-0-13-92:~/rancher# 

```
**Expected Result**

should install successfully and fetch the right image

**Screenshots**

**Additional context**

**Comment by alexandreLamarre** [comment]
There's also a missing version bump for prometheus-node-exporter that should be included (1.3.1 -> 1.7.0)

**Comment by pankajsqe** [comment]
Issue has been fixed and verified on Air-gap setup. Rancher Monitoring Prometheus Node Exporter pod in the cattle-monitoring-system namespace pulling correct image.


![image (2)](https://github.com/user-attachments/assets/caa0e0bd-306d-4811-9789-e7b350778707)
